cpu/esp_common/Makefile.include

```diff
@@ -0,0 +1,216 @@
+# Default compile configurations
+
+# FLASH_MODE=[ dout | dio | qout | qio ]
+# use flash mode dout by default to keep GPIO9 and GPIO10 free for use
+FLASH_MODE ?= dout
+
+# SPECIAL module dependencies
+# cannot be done in Makefile.dep since Makefile.dep is included too late
+
+# use esp_now as default netdev if no other netdev module is enabled
+ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
+  ifeq (,$(filter esp_wifi esp_eth,$(USEMODULE)))
+    USEMODULE += esp_now
+  endif
+endif
+
+ifneq (,$(filter esp_gdbstub,$(USEMODULE)))
+  USEMODULE += esp_gdb
+endif
+
+ifneq (,$(filter log_color,$(USEMODULE)))
+  USEMODULE += esp_log_colored
+endif
+
+ifneq (,$(filter 4 5,$(LOG_LEVEL)))
+  USEMODULE += esp_log_startup
+endif
+
+# ESP* pseudomodules
+
+PSEUDOMODULES += esp_gdb
+PSEUDOMODULES += esp_i2c_sw
+PSEUDOMODULES += esp_log_colored
+PSEUDOMODULES += esp_log_tagged
+PSEUDOMODULES += esp_log_startup
+PSEUDOMODULES += esp_qemu
+PSEUDOMODULES += esp_spiffs
+PSEUDOMODULES += esp_wifi_any
+
+# Modules used by ESP*
+
+USEMODULE += esp_common
+USEMODULE += esp_common_periph
+USEMODULE += esp_freertos
+USEMODULE += esp_idf
+USEMODULE += log
+USEMODULE += newlib
+USEMODULE += periph
+USEMODULE += periph_hwrng
+USEMODULE += periph_flash
+USEMODULE += periph_uart
+USEMODULE += random
+USEMODULE += stdio_uart
+USEMODULE += xtensa
+
+# Common includes
+
+INCLUDES += -I$(RIOTBOARD)/common/$(CPU)/include
+INCLUDES += -I$(RIOTCPU)/esp_common
+INCLUDES += -I$(RIOTCPU)/esp_common/include
+INCLUDES += -I$(RIOTCPU)/esp_common/include/freertos
+INCLUDES += -I$(RIOTCPU)/esp_common/vendor/
+INCLUDES += -I$(RIOTCPU)/esp_common/vendor/esp
+
+# Flags
+
+CFLAGS += -Wno-unused-parameter -Wformat=0
+CFLAGS += -mlongcalls -mtext-section-literals -fstrict-volatile-bitfields
+CFLAGS += -fdata-sections -ffunction-sections -fzero-initialized-in-bss
+
+OPTIONAL_CFLAGS_BLACKLIST += -Wformat-overflow
+OPTIONAL_CFLAGS_BLACKLIST += -Wformat-truncation
+OPTIONAL_CFLAGS_BLACKLIST += -gz
+
+ASFLAGS += --longcalls --text-section-literals
+
+# thin archives trigger a reboot loop - see #12258, #12035, #12346
+ARFLAGS = rcs
+
+CFLAGS_DBG ?= -ggdb -g3
+CFLAGS_OPT ?= -Os
+
+# override default CFLAGS_OPT in case of enabled module esp_gdb
+ifneq (,$(filter esp_gdb,$(USEMODULE)))
+  CFLAGS_OPT = -Og
+endif
+
+CFLAGS += $(CFLAGS_OPT) $(CFLAGS_DBG)
+
+# add -DQEMU for qemu
+ifneq (,$(filter esp_qemu,$(USEMODULE)))
+  CFLAGS += -DQEMU
+endif
+
+# use 32 priority levels if any WiFi interface or the ETH interface is used
+ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
+  CFLAGS += -DSCHED_PRIO_LEVELS=32
+endif
+
+ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
+  INCLUDES += -I$(RIOTCPU)/esp_common/esp-wifi
+endif
+
+# lwIP thread priority has to be lower than that of ESP hardware driver threads
+ifneq (,$(filter lwip,$(USEMODULE)))
+  CFLAGS += -DTCPIP_THREAD_PRIO=5
+endif
+
+# if SPI RAM is enabled, the ESP-IDF heap and quot flash mode have to be used
```

Comment:
add "as well" at the end of the sentence?

Reply:
I had to correct the comment completely.